### ISSUE TITLE
Preffered unsupported browser users can't use Station

### ISSUE THREAD

**Issue by longgamma-massa** [massalabs/station#1170]
**Context**
Massa Station requirements for Mainnet were to work on Chromium and Firefox browsers. This is respected.
However, users for whom preferred browser is Safari see an empty screen (err) - can't run station there. 

**User flow**
- Install station
- Open the app (double-click or via icon-tray)
- Safari started -> white empty page shown.

**How to**
2 first solutions for this, until Safari becomes a priority:
1. On the Station's landing page, add supported browsers: Chromium and Firefox. @LucasF243 
2. For users whose browser is one of unsupported ones, open Massa station in a window of supported browser. - can't be done at the moment. During 1-day investigation we couldn't find the solution

**Technical details**
*Give the technical insights so anyone in the team can tackle the tasks - Dev*

**QA testing**
*Does this task require some QA tests ?*
*If yes, explain how to validate it*

**Comment by longgamma-massa** [comment]
<img width="676" alt="image" src="https://github.com/massalabs/station/assets/113043914/316c4d74-b19e-498e-9369-c78791c7a052">

**Comment by SlnPons** [comment]
@thomas-senechal or @Thykof do you have a clear plan to address this issue?

**Comment by Thykof** [comment]
> @thomas-senechal or @Thykof do you have a clear plan to address this issue?

We won't address this issue for now, @thomas-senechal investigate and decided with @longgamma-massa to not do it. @thomas-senechal can you explain what you found and why there is no viable solution, then close this issue, please :) ?

**Comment by thomas-senechal** [comment]
> We won't address this issue for now, @thomas-senechal investigate and decided with @longgamma-massa to not do it. @thomas-senechal can you explain what you found and why there is no viable solution, then close this issue, please :) ?

As far as I remember, we wanted to do solution 2 from the issue description:
> For users whose browser is one of unsupported ones, open Massa station in a window of supported browser

But to do that, we need to do the following things:
- Find a way to get the list of web browsers
- Either:
  - Find a way to know the type of each web browsers -> For example, Brave is Chromium based, so it's supported
  - Set a list of supported browser -> That means that web browsers outside of the list won't be supported
- Once we have the list of supported and installed web browsers, we can ask user to select one (or select the latest used -> how to do that ?)
- We might face issues with how to start a web browser with a given URL and other stuff like that

In summary, that would require a lot of dev time and might be unstable at the begining because of how web browsers work and compatibility issues
Also, it's not worth to spend days on an issue that only a few people are facing (only people that are using Safari as their main web browser, or non Chrome based web browser)

And so, the best solution was solution 1, to simply tell people which that we only support Chromium Based and Firefox web browsers on the landing page

**Comment by SlnPons** [comment]
Sounds good, thanks.
